packages/easy-email-core/src/utils/TemplateEngineManager.tsx

```diff
@@ -51,7 +51,7 @@ function generateConditionTemplate(
       (isNumber(condition.right) ? condition.right : `"${condition.right}"`)
     );
   };
-  const uuid = nanoid(5);
+  const uuid = uuidv4();
   const variables = groups.map((_, index) => `con_${index}_${uuid}`);

```

Comment:
uuid is to prevent the condition from being overwritten, if the condition is the same, it will not be affected.In contrast, I prefer object-hash
```ts
import hash from 'object-hash';
const uuid = hash(condition) 

```

Reply:
I think I'll need more info to understand your needs. I thought you needed a random id being generated for all entries (yes the uuidV4 is a little bit longer than `nanoid` but it should still return a randomized string). Are you saying you would not need any random string?

![image](https://user-images.githubusercontent.com/6645382/224345096-5f29291c-31ee-4b83-bfc2-94c4b92f7768.png)

Reply:
Yes, I need a unique ID to ensure that variables are not overwritten. If they are the same, they can be overwritten, for example.
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/20615214/224484629-683e2d76-476a-4e63-a5f5-464812202f0c.png">

Reply:
Yup. I understood that. Can you explain to me what is not working for you with this library change (which is essentially generating a unique ID as well)?